.github/workflows/ci.yml

```diff
@@ -2,8 +2,6 @@ name: nf-core CI
 # This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
 on:
   push:
-    branches:
-      - dev
```

Comment:
Why this condition being removed? We want the small test to be run for every commit to the dev branch.

Reply:
removed it due to it halts the github pipeline when checking in